internal/statemachine/helper.go

```diff
@@ -407,11 +407,15 @@ func maxOffset(offset1, offset2 quantity.Offset) quantity.Offset {
 	return offset2
 }
 
-// setStructureSize sets both Size and MinSize to the same value on a structure
+// setStructureSize raise both Size and MinSize to at least the given size
 // It helps make sure whatever value is used in snapd, it is set to the size we need
 func setStructureSize(s *gadget.VolumeStructure, size quantity.Size) {
```

Comment:
All good as is, but maybe for clarity the name could be adjusted (especially that it's used internally only). Since this no longer sets sizes, but sets the 'highest' from the sizes.

Reply:
Good catch! I renamed it.